perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GeneralServiceManagerMethod.java

```diff
@@ -21,7 +21,8 @@
 	 */
 	blockServiceOnFacility {
```

Comment:
Please copy-paste all methods from here (before adding ``@deprecated`` to docs) to ``ServicesManagerMethod``, since you already changed CLI (client side) and new API would be missing. Also like this we can allow clients to switch to new API and then later remove this one.